blackbox/docs/config/environment.rst

```diff
@@ -126,3 +126,21 @@ There are various environment variables to control garbage collection logging
 for CrateDB.
 
 See :ref:`conf-logging-gc`.
+
+``CRATE_HEAP_DUMP_PATH``
+========================
+
+CrateDB will create a heap dump in case of a crash caused by an out of memory
+error. It is necessary to make sure that there is enough disk space available
+so that this heap dump could be created. The location of this heap dump can be
+set via the ``CRATE_HEAP_DUMP_PATH`` environment variable.
+
+:CRATE_HEAP_DUMP_PATH:
+  | Path to a directory or file where the heap dump will be created. If a
```

Comment:
Pipe makes an implicit line break. I think this is not necessary for the 1st paragraph, only for the lines with the default values.